Documentation.md

```diff
@@ -164,590 +174,606 @@ and made available to your JS client code.
 * You have the option to show or hide the pre-built UI by showing or hiding the
 `containerDiv` into which you place the iframe, or applying a CSS rule like
 this:
-```
+```css
 .containerDiv iframe {
-   display: none;
+  display: none;
 }
 ```
 * The pre-built CCP UI is portrait oriented and capable of contracting
-  horizontally to fit smaller widths.  It can expand from a width of 200px to
-  a maximum of 320px based on the size of its container div.  If the CCP is
+  horizontally to fit smaller widths. It can expand from a width of 200px to
+  a maximum of 320px based on the size of its container div. If the CCP is
   placed into a container where it would be sized under 221px in width, the CCP
   switches to a different layout style with smaller buttons and fonts.
-* In its normal larger style, the CCP is 465px tall.  In its smaller form, the
+* In its normal larger style, the CCP is 465px tall. In its smaller form, the
   CCP is reduced to 400px tall.
 * CSS styles you add to your site will NOT be applied to the CCP because it is
   rendered in an iframe.
 * If you are trying to use chat specific functionalities, please also include
   [ChatJS](https://github.com/amazon-connect/amazon-connect-chatjs) in your code.
-  We omit ChatJS from the Makefile so that streams can be used without ChatJS. 
+  We omit ChatJS from the Makefile so that streams can be used without ChatJS.
   Streams only needs ChatJS when it is being used for chat. Note that when including ChatJS,
-  it must be imported after StreamsJS, or there will be AWS SDK issues 
+  it must be imported after StreamsJS, or there will be AWS SDK issues
   (ChatJS relies on the ConnectParticipant Service, which is not in the Streams AWS SDK).
 
-### Event Subscription
+### `connect.core.terminate()`
+
+Unitializes Amazon Connect Streams. Removing any subscription methods that have been called.
+
+The CCP iframe will not be removed though, so you'll have to manually remove it.
+
+## Event Subscription
 Event subscriptions link your app into the heartbeat of Amazon Connect by allowing your
 code to be called when new agent information is available.
 
 Event subscription works by providing callbacks to the Streams API which are
 called when the agent is initialized, and when contacts are first detected.
 Then, `on*()` event subscription methods are provided with callbacks which are
-called when events occur on those specific objects.  These return subscription
+called when events occur on those specific objects. These return subscription
 objects, which for contacts are automatically cleaned up when those contacts no
-longer exist.  Users can also manually unsubscribe from events by calling
+longer exist. Users can also manually unsubscribe from events by calling
 `sub.unsubscribe()` on the subscriptions returned by these methods.
 
 ### `connect.agent()`
+```js
+connect.agent(function(agent) { /* ... */ });
```

Comment:
**Nitpick:** Replaced all `...` ocurrences with `/* ... */` so that it's syntactically-valid JavaScript.